public/sass/components/_dashboard_grid.scss

```diff
@@ -96,10 +96,12 @@
 }
 
 // Hack for preventing panel menu overlapping.
+.react-grid-item.context-menu-open,
+.react-grid-item.resizing,
```

Comment:
this fixes a regression where a panel that's being resized may temporarily go under an adjacent hovered panel when the mouse moves fast.